SQLQueryGroupProjectTeamB1Julia.sql

```diff
@@ -0,0 +1,255 @@
+				JOIN tblHOUR H
+					ON H.HourID = S.BeginHourID
+				JOIN tblCOMPENSATION C
+					ON C.CompID = S.CompID
+				JOIN tblSHIFT_TYPE ST
+					ON ST.ShiftTypeID = S.ShiftTypeID
+				JOIN tblLOCATION L
+					ON L.LocationID = S.LocationID
+			 WHERE  Q.QuartName = @QuarterName
+				AND M.MonthNum = @MonthNumber
+				AND D.[DayName] = @DayNumber
+				AND S.BeginHourID = @BeginHour
+				AND S.EndHourID = @EndHour
+				AND L.LocationName = @Location
+				AND C.CompName = @Compensation
+				AND ST.ShiftTypeName = @ShiftTypeName
+				AND S.[YEAR] = @Year)
+
+SET @LOC_ID = (SELECT L.LocationID
+			   FROM tblLOCATION L
+			   WHERE L.LocationName = @Location)
+SET @Q_ID = (SELECT Q.QuarterID
+			 FROM tblQUARTER Q
+			 WHERE Q.QuartName = @QuarterName)
+SET @MON_ID = (SELECT M.MonthID
+			   FROM tblMONTH M
+			   WHERE M.MonthNum = @MonthNumber)
+SET @DAY_ID = (SELECT D.DayID
+			   FROM tblDAY D
+			   WHERE D.[DayName] = @DayNumber)
+SET @BH_ID = (SELECT H1.HourID
+			  FROM tblHOUR H1
+			  WHERE H1.HourName = @BeginHour)
+SET @EH_ID = (SELECT H2.HourID
+			  FROM tblHOUR H2
+			  WHERE H2.HourName = @EndHour)
+SET @COMP_ID = (SELECT C.CompID
+				FROM tblCOMPENSATION C
+				WHERE C.CompName = @Compensation)
+SET @ST_ID = (SELECT ST.ShiftTypeId
+			  FROM tblSHIFT_TYPE ST
+			  WHERE ST.ShiftTypeName = @ShiftTypeName)
+
+INSERT INTO tblSHIFT(ShiftID, LocationID, QuarterID, MonthID, DayID, [YEAR], BeginHourID, EndHourID, CompID, ShiftTypeID)
+VALUES (@S_ID, @LOC_ID, @Q_ID, @MON_ID, @DAY_ID, @Year, @BH_ID, @EH_ID, @COMP_ID, @ST_ID)
+
+--Write the stored procedure to assign a new pronoun to an existing pronoun set.
+
+CREATE PROCEDURE SP_NewPronoun
+@PronounTitle VARCHAR(50),
+@PronounSetName VARCHAR(50),
+@PronounDescription VARCHAR(500)
+
+AS
+
+DECLARE @PRO_ID INT, @PS_ID INT
+
+SET @PS_ID = (SELECT PS.PronounSetID
+			  FROM tblPRONOUN_SET PS
+			  WHERE PS.PSetName = @PronounSetName)
+
+INSERT INTO tblPRONOUN(PronounTitle, PronounsDescr)
+VALUES(@PronounTitle, @PronounDescription)
+
+SET @PRO_ID = SCOPE_IDENTITY()
+
+INSERT INTO tblPRONOUN_PRONOUN_SET(PronounSetID, PronounID)
+VALUES (@PS_ID, @PRO_ID)
+
+--POPULATE TABLES
+
+--Populate the following look-up tables: tblCOMPENSATION, tblQUARTER, tblHOUR
+
+INSERT INTO tblCOMPENSATION(CompName, CompTypeDescr, CompAmount)
+VALUES('Winter or Spring Break', 'Compensation given for taking a shift during Winter or Spring Break', 50),
+		('Holiday Comp', 'Compensation given for taking a shift on a holiday', 75),
+		('Birthday Comp', 'Compensation given for taking a shift on your birthday', 100),
+		('Finals Week Comp', 'Compensation given for taking a shift during finals week', 125),
+		('No Comp', 'No compensation given', 0)
+
+
+INSERT INTO tblQUARTER(QuartName, QuartBegin, QuartEnd, QuartDescr)
+VALUES ('Winter', '01-06-2020', '03-13-2020', 'Winter quarter is the quarter in the beginning of the new year - it is usually the second quarter of the new school year - the weather can be rainy or snowy'),

```

Comment:
since the group decided to removed the date columns from tblQUARTER, please remove date parameters from these insert statements